src/components/Header.js

```diff
@@ -25,81 +25,80 @@ const handleLogout = () => {
 
   return (
     <div className='header'>
-      <h1 className='heading'>Welcome! Find Your Book Besties Today!</h1>
-        <Navbar {...args}>
-          <div>
-          <NavbarBrand  href="/" className='navbar-brand'>
-          <img 
-              className='logo'
-              alt="logo"
-              src={logo}
-            />
-          </NavbarBrand>
-          </div>
-          <div className='dropDownMenu'>
-          <NavbarToggler  onClick={toggle} className='navbar-toggler'/>
-          <Collapse isOpen={isOpen} navbar>
-            <Nav navbar className='navbar-nav'>
-              <NavItem className='nav-item'>
-                <NavLink to="/" className='custom-link'>Home</NavLink>
-              </NavItem>
-              { isLoggedIn && (
-                <>
+      <Navbar {...args}>
+        <div>
+        <NavbarBrand  href="/" className='navbar-brand'>
```

Comment:
Internal navigation (any links within React) should only ever come from react router with the `to` attribute.